xinterpreters/xinterpreters/host/tests/test_smoke/test_smoke.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# Copyright (c) 2020, XMOS Ltd, All rights reserved
+
+import sys, os
+import numpy as np
+import cv2
+
+from xinterpreters import xcore_tflm_host_interpreter
+
+ie = xcore_tflm_host_interpreter()
+ie.set_model(model_path="./smoke_model.tflite", params_path="./smoke_model.flash")
+with open("./detection_0.raw", "rb") as fd:
```

Comment:
This change got left out I think, https://github.com/xmos/lib_tflite_micro/commit/38e80e03ec7425ac85fd3a79879b86b59553be2b